libs/store/src/lib/loggers/race.ts

```diff
@@ -0,0 +1,16 @@
+import { AppErrorCode, AppMessageLogType } from '@razor/models';
+import { Dispatch } from '../store';
+
+export const raceNotFound = async (
+  dispatch: Dispatch,
+  rid: string,
+  additionalMessage?: string,
+) => {
+  dispatch.game.sendLogMessage({
+    message: `Race with id ${rid} does not exist`,
+    code: AppErrorCode.RaceNotExists,
+    related: additionalMessage ? additionalMessage : '',
+    type: AppMessageLogType.Error,
+  });
+  return;
```

Comment:
this statement is redundant.

Reply:
Fixed in PR #14 under commit a30c31abb7742dfe160b204a5bca3285d6a0d65a